src/components/draw/DrawDirective.js

```diff
@@ -110,6 +110,9 @@ goog.require('ga_permalink');
       var updateHelpTooltip = function(overlay, type, drawStarted, onFirstPoint,
           onLastPoint) {
         var helpMsg = $translate.instant('draw_start_' + type);
+        if (type == 'marker' || type == 'annotation') {
+          helpMsg = $translate.instant('draw_next_' + type);
```

Comment:
On the screenshot of Cedric the text not translated is 'draw_start_marker' . The draw next next is not used when drawing annotation and marker.